### PR TITLE
Withdocs configurable

### PR DESCRIPTION
**Pull request by jptredoux**
Adresses: https://github.com/tuchk4/storybook-readme/issues/33

### CODE REVIEW

**Comment by tuchk4** [comment]
@jptredoux Can you please add additional example with custom Preview component?

File: README.md [hunk 1]
```diff
@@ -67,13 +67,13 @@ storiesOf('Button', module)
...
 storiesOf('Content', module)
-  .add('Default', withDocs(ButtonReadme, () => <Content>Hello Button<Content/>))
+  .add('Default', withDocs()(ButtonReadme, () => <Content>Hello Button<Content/>))
 ```
 
 #### Use as Higher Order Component
 
 - *withReadme(readme, story)*
-- *withDocs(readme, story)*
+- *withDocs(configuration)(readme, story)*
```
Comment (tuchk4) [inline README.md]: add `configuration` details. What can we configure with it?

**Comment by tuchk4** [comment]
@jptredoux ping

**Comment by jptredoux** [comment]
@tuchk4 : thanks for the feedback -- i'll be able to make the requested changes this weekend 👍

**Comment by tuchk4** [comment]
@jptredoux ping :)

**Comment by tuchk4** [comment]
Implemented at 5eac4ddb3125526c7cf9682f0fbe580c3dfcd110

**Pull request closed without merging**